### PR TITLE
implemented testVideoUrl, fixed PR's issues, changed podspec and mani…

### PR DESCRIPTION
**Pull request by TolyaAfanasev**
…fest files.

### CODE REVIEW

File: TelevisionAcademyIOSPlayerPlugin.podspec [hunk 1]
```diff
@@ -15,10 +15,10 @@ Pod::Spec.new do |s|
     s.requires_arc = true
     s.swift_version = '5.0'
     s.static_framework = true
-    s.resources = ['ios/TelevisionAcademyIOSPlayerPlugin/Resources/Images/*.png', 'ios/TelevisionAcademyIOSPlayerPlugin/PluginClasses/*.{xib,nib,storyboard}']
-    s.source_files = 'ios/TelevisionAcademyIOSPlayerPlugin/PluginClasses/**/*.{swift,h,m}'
```
Comment (PRuedaApplicaster) [inline TelevisionAcademyIOSPlayerPlugin.podspec]: You will need to look recursvely in the folders, as not all your classes are under PluginClasses (they are in subfolders).

File: ios/TelevisionAcademyIOSPlayerPlugin/PluginClasses/BitmovinPlayerPlugin.swift [hunk 2]
```diff
@@ -40,7 +38,13 @@ extension BitmovinPlayerPlugin: ZPPlayerProtocol {
 
     public static func pluggablePlayerInit(playableItems items: [ZPPlayable]?, configurationJSON: NSDictionary?) -> ZPPlayerProtocol? {
 
-        guard let videos = items else { return nil }
+        var videos: [ZPPlayable] = items ?? [ZPPlayable]()
+
+        if let d = configurationJSON,
```
Comment (PRuedaApplicaster) [inline ios/TelevisionAcademyIOSPlayerPlugin/PluginClasses/BitmovinPlayerPlugin.swift]: Please name it something more readable, not d :)

File: ios/TelevisionAcademyIOSPlayerPlugin/PluginClasses/BitmovinPlayerPlugin.swift [hunk 3]
```diff
@@ -40,7 +38,13 @@ extension BitmovinPlayerPlugin: ZPPlayerProtocol {
...
+            let testUrl = d["test_video_url"] as? String {
```
Comment (PRuedaApplicaster) [inline ios/TelevisionAcademyIOSPlayerPlugin/PluginClasses/BitmovinPlayerPlugin.swift]: You should check also that the string is not empty. We don't want to do this if the string is "".

File: ios/plugin-manifest.json [hunk 4]
```diff
@@ -1,10 +1,10 @@
 {
   "api": {
     "require_startup_execution": false,
-    "class_name": "TelevisionAcademyIOSPlayerPlugin.BitmovinPlayerPlugin"
+    "class_name": "BitmovinPlayerPlugin"
```
Comment (PRuedaApplicaster) [inline ios/plugin-manifest.json]: You will need the prefix.

File: TelevisionAcademyIOSPlayerPlugin.podspec (hunk 1 above)
Comment (TolyaAfanasev) [reply TelevisionAcademyIOSPlayerPlugin.podspec]: Added subfolders /**/*.swift

**Pull request merged**